Sources/NIO/Interfaces.swift

```diff
@@ -206,6 +227,57 @@ extension NIONetworkDevice {
         /// This constructor will fail if NIO does not understand the format of the underlying
         /// socket address family. This is quite common: for example, Linux will return AF_PACKET
         /// addressed interfaces on most platforms, which NIO does not currently understand.
+#if os(Windows)
+        internal init?(_ pAdapter: UnsafeMutablePointer<IP_ADAPTER_ADDRESSES>,
+                       _ pAddress: UnsafeMutablePointer<IP_ADAPTER_UNICAST_ADDRESS>) {
```

Comment:
Do we need these pointers mutable? If not it'd be nice to ask for them `const`. It's much harder to walk that back later.

Reply:
I suppose that we can construct the `UnsafePointer` from the `UnsafeMutablePointer`.  They must be `UnsafeMutablePointer` for the call into the Windows SDK AFAIK.

Reply:
Woof, really? If they do then we should leave them as is.

Reply:
Yes, the Windows headers don't very often mark inputs as `const` so we need things to be mutable.